src/parser/parser_utils.js

```diff
@@ -154,48 +154,25 @@ export class ParserUtils {
    * @return {Array}
    */
   splitVAST(ads) {
-    const splittedVAST = [];
-    let adPod = [];
-    let isAdPod = false;
-    let adPodSequence = null;
-
-    ads.forEach(ad => {
-      // Check if we are at the beginning of a new AdPod
-      if (ad.sequence === 1) {
-        // Check if we were already in the middle of an AdPod
-        // If so push the previous AdPod to the splittedVAST
-        if (isAdPod) {
-          splittedVAST.push(adPod);
-        }
-        // Clean up and start storing the new AdPod
-        adPod = [];
-        adPod.push(ad);
-        adPodSequence = 1;
-      } else {
-        // Check if we were already in the middle of an AdPod
-        if (isAdPod) {
-          // Check if the current ad is the next ad of the AdPod
-          if (ad.sequence === adPodSequence + 1) {
-            adPod.push(ad);
-            adPodSequence = ad.sequence;
-          } else {
-            // If the current ad is not the next in the AdPod and not the first of an AdPod,
-            // push the previous AdPod and the current ad to the splittedVAST and clean up
-            splittedVAST.push(adPod);
-            splittedVAST.push([ad]);
-            isAdPod = false;
-            adPodSequence = null;
-            adPod = [];
+    const pods = [];
```

Comment:
we can call this `splittedVAST` since semantically it's an array of ad and adPods